ios/browser/api/bookmarks/brave_bookmarks_api.mm

```diff
@@ -8,7 +8,10 @@
 #include "base/compiler_specific.h"
 #include "base/containers/adapters.h"
 #include "base/guid.h"
+#include "base/sequenced_task_runner.h"
```

Comment:
I believe we do not need to include  sequenced task runner.

Reply:
Needed for this: https://bravesoftware.slack.com/archives/C7VLGSR55/p1621246730369800